### PR TITLE
ExposePatchedMethods v2.0

### PR DESCRIPTION
**Pull request by kazu0617**
Make configratable to show everywhare.

Make buildable for Linux, more shrink.

### CODE REVIEW

**Comment by kazu0617** [comment]
If you want to test, use this dlls.(GitHub doesn't allow to upload dll directly, so shrink to gzip.)

ExposePatchedMethods.dll and zzExposePatchedMethods.dll is same.

[ExposePatchedMethods.tar.gz](https://github.com/EIA485/NeosExposePatchedMethods/files/8363244/ExposePatchedMethods.tar.gz)


---- Checksum ----
名前: ExposePatchedMethods.tar.gz
サイズ: 3469 バイト (3 KiB)
CRC32: 9746CE33
CRC64: D9BDE23B4545A99A
SHA256: 9ADC39EC9FB5843C096D11C8CD4DA8974D15486038A779FD1DFCD37FEFE5793C
SHA1: 1DFBAD5B28AF5219CC0FC3168CA39AFD92008CE9
BLAKE2sp: 4ADD70102D8B0BE5B4C699418717F3B74F185EB5ABE0A3DF2358D2C9F39059F6

名前: ExposePatchedMethods.dll
サイズ: 7168 バイト (7 KiB)
CRC32: 1D87785D
CRC64: FAD8C2AE75C92C22
SHA256: BFBDBB76DE607D109662E21CB7BB9B4EE962343DC1DBE0E00E0D821484B3D99D
SHA1: 0C0A18C4BB4921734D0B7C542130F8C80AC19FC4
BLAKE2sp: AB3C69478FE0904C1A80601939706915286CE38CE0F8D9D63A08F640A0C78353

名前: ExposePatchedMethods.json
サイズ: 69 バイト
CRC32: B0CC74E6
CRC64: 489A0916FC9520D7
SHA256: 7F10FFD834218A01E7B7CE616FC4BA66B2BC94A8F598DD53847B813D296EB9DD
SHA1: 139E533818E562F1956AA30FEC22884F0D210CA2
BLAKE2sp: 32DCD7F26C73D3A659A613618000344C7C89DCEB6BA00D87685A43B2B1641A1D
---- Checksum ----

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
+A [NeosModLoader](https://github.com/zkxs/NeosModLoader) mod for [Neos](https://neos.com/) that exposes what mods you have loaded to your userspace. it does this by creating a slot called "Loaded mods" with its children being all of the harmony ids that had been used to patch a method at the time this mod was called. mods are called in mod file name alphabetical order. if you want this mod to exposes all mods then prefix it's file name with zz. if you also want to expose what mods you have loaded under your userroot you can enable key "Show Everywhare" in Config.
```
Comment (l-2-j) [inline README.md]: Small nitpick about spelling, "Everywhare" -> "Everywhere". Also applies to the Config key.

File: ExposePatchedMethods.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+using System.Collections.Generic;
+
+namespace ExposePatchedMethods
+{
+	public class ExposePatchedMethods : NeosMod
+	{
+        public override string Name => BuildInfo.Name;
+        public override string Author => BuildInfo.Author;
+        public override string Version => BuildInfo.Version;
+        public override string Link => BuildInfo.Link;
```
Comment (EIA485) [inline ExposePatchedMethods.cs:14]: hey, thanks for thanks for the pr. can you revert this style change?

**Comment by EIA485** [comment]
can you revert all the style changes? like moving the files around

**Comment by kazu0617** [comment]
image
np. this style is based ljoonal style, and all of my repo just using this style. if you unlike it, just revert plz.

**Comment by EIA485** [comment]
sorry about that, did not realize that's how the edit feature works. il pr your repo in a sec with a revert

**Comment by EIA485** [comment]
> image np. this style is based ljoonal style, and all of my repo just using this style. if you unlike it, just revert plz.

sorry about that, did not realize that's how the edit feature works. il pr your repo in a sec with a revert

**Pull request merged**